src/redisai.c

```diff
@@ -583,8 +583,9 @@ int RedisAI_ScriptRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv
     if (argc < 6)
         return RedisModule_WrongArity(ctx);
 
-    // Convert The script run command into A DAG command that contains a single op.
-    return ProcessRunCommand(ctx, argv, argc, CMD_SCRIPTRUN, REDISAI_DAG_WRITE_MODE);
+    // Convert The script run command into a DAG command that contains a single op.
+    bool ro_dag = false;
+    return RedisAI_ExecuteCommand(ctx, argv, argc, CMD_SCRIPTRUN, ro_dag);
```

Comment:
you don't have to declare the variable - you can just send it in the function
leave or modify at your discretion.